### PR TITLE
git: udpate to v2.39.1 and add git-lfs (#5528)

### LINKED ISSUE #5528: [Package Request] add Git LFS to the Git package
### Software Name

Git

### Brief Description

Git LFS (Large File Storage) is an extension for Git that handles large files.

It would be nice to either include this with the current Git package or add it as a separate package. I have tested that it works on my synology with the existing Git package using the binary released [here](https://github.com/git-lfs/git-lfs/releases/tag/v3.3.0)

### Website

https://git-lfs.com/

### Documentation

https://github.com/git-lfs/git-lfs/tree/main/docs?utm_source=gitlfs_site&utm_medium=docs_link&utm_campaign=gitlfs

### Build/Installation documentation

https://github.com/git-lfs/git-lfs/blob/main/README.md

### Source code

https://github.com/git-lfs/git-lfs

### OpenSource License

MIT

### Other License

_No response_

### PR DESCRIPTION
**Pull request by hgy59**
## Description

Closes #5528

## Checklist

- [x] Build rule `all-supported` completed successfully
- [x] New installation of package completed successfully
- [x] Package upgrade completed successfully (Manually install the package again)
- [x] Package [functionality was tested](https://github.com/SynoCommunity/spksrc/wiki/Package-Update-Policy#tests-checks)
- [ ] Any needed [documentation](https://github.com/SynoCommunity/spksrc/wiki/Create-documentation) is updated/created

### Type of change

- [x] Package update

### CODE REVIEW

**Comment by hgy59** [comment]
@th0ma7 please have a look for the solution to add additional features to cross/curl (for tvheadend).

File: cross/tvheadend/Makefile [hunk 1]
```diff
@@ -11,7 +11,12 @@ PKG_DIR = $(PKG_NAME)-$(PKG_GIT_HASH)
 DEPENDS  = cross/openssl cross/gnutls
 DEPENDS += cross/uriparser cross/pcre2 cross/libiconv cross/fontconfig
 DEPENDS += cross/libhdhomerun cross/libdvbcsa cross/dvb-apps cross/pngquant
+DEPENDS += cross/libwebsockets
```
Comment (ymartin59) [inline cross/tvheadend/Makefile:14]: File not related to git, please move changeset to another PR

File: diyspk/curl/Makefile [hunk 2]
```diff
@@ -0,0 +1,23 @@
+SPK_NAME = curl
```
Comment (ymartin59) [inline diyspk/curl/Makefile:1]: File not related to git, please move changeset to another PR

**Review by ymartin59 — CHANGES_REQUESTED** [review]
Please move out the two intruder files. Except these, LGTM

**Comment by hgy59** [comment]
> @th0ma7 please have a look for the solution to add additional features to cross/curl (for tvheadend).

Theupdate of curl/tvheadend is moved to PR #5543

**Pull request merged**